### PR TITLE
Add an indestructibleBuildables cvar

### PR DESCRIPTION
**Pull request by necessarily-equal**
This is used in the juggernaut mod

### CODE REVIEW

File: src/sgame/sg_buildable.cpp [hunk 1]
```diff
@@ -1852,6 +1856,9 @@ static gentity_t *SpawnBuildable( gentity_t *builder, buildable_t buildable, con
 	built->enabled = false;
 	built->spawned = false;
 
+	built->flags = g_indestructibleBuildables.Get() ?
+		FL_GODMODE | FL_NOTARGET : 0;
```
[the remark this answers is not in the document]
Comment (necessarily-equal) [reply src/sgame/sg_buildable.cpp:1860]: It seems it means "not attacked by turrets, bots, acid tubes, etc"

**Comment by bmorel** [comment]
Note that there is a way to test this outside of jug mod or whatever: `/devmap plat23` then `/god; kill`

**Comment by bmorel** [comment]
This does not work here.

**Comment by slipher** [comment]
LGTM

**Comment by bmorel** [comment]
Weird, I checked anew and it worked, this time. I probably had something wrong the 1st time.
LGTM.

**Pull request merged**